module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderText.kt

```diff
@@ -1,28 +1,68 @@
 package org.cru.godtools.shared.renderer.content
 
 import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.text.font.FontStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import coil3.compose.AsyncImage
+import coil3.compose.LocalPlatformContext
+import coil3.request.ImageRequest
 import com.github.ajalt.colormath.extensions.android.composecolor.toComposeColor
 import org.cru.godtools.shared.renderer.content.extensions.textAlign
+import org.cru.godtools.shared.renderer.content.extensions.toPath
 import org.cru.godtools.shared.renderer.content.extensions.visibility
 import org.cru.godtools.shared.renderer.state.State
+import org.cru.godtools.shared.renderer.util.LocalResourceFileSystem
+import org.cru.godtools.shared.tool.parser.model.Resource
 import org.cru.godtools.shared.tool.parser.model.Text
 
 @Composable
-internal fun ColumnScope.RenderText(text: Text, state: State) = RenderTextNode(
-    text,
-    Modifier
+internal fun ColumnScope.RenderText(text: Text, state: State) = Row(
+    modifier = Modifier
         .visibility(text, state)
         .padding(horizontal = Horizontal_Padding)
-        .fillMaxWidth(),
-)
+        .fillMaxWidth()
+) {
+    val imagePaddingToText: Int = 10
+
+    text.startImage?.let {
+        RenderImageNode(
+            resource = it,
+            imageSize = text.startImageSize,
+            modifier = Modifier
+                .align(Alignment.CenterVertically)
+                .padding(paddingValues = PaddingValues(end = imagePaddingToText.dp))
+        )
+    }
+
+    RenderTextNode(
+        text = text,
+        modifier = Modifier
+            .align(Alignment.CenterVertically)
```

Comment:
for now let's do `.align(Alignment.Top)` to match how Android is currently rendering these.

This can be changed if there is feedback about how the product team wants the alignment of the text to work